docs/plugin/interfaces/tags.md

```diff
@@ -564,9 +569,12 @@ For more details, click [here](../interfaces/#asynchronous-functions)
 
 #### @encode
 
-This tag encodes data into an alternate format.
+This tag encodes or decodes (if an input parameter) data into/from an alternate format.
 
-* `@encode:base64` encodes arrays as base64 JSON-RPC arrays, on the condition that the array base is type `uint8_t`.
+* `@encode:base64` encodes/decodes arrays (or std::vector or iterator) as base64 JSON-RPC string, on the condition that the array base is type `uint8_t`.
```

Comment:
encode:hex encode:base64 work for C-style arrays, std::vector and ptr+len buffers if the type of the array/vector is uint8_t or char. They do not work yet for iterators.